### ISSUE TITLE
Glossary - Weird reading order? (probably missing labels) [2.21 RDI report]

### ISSUE THREAD

**Issue by MeD-Owner** [MeD-DMC/Adapt-accessibilityfixes#68]
While we navigate the glossary there is information presented, but the experience is not the same for all users. 

When tabbing in the Glossary, JAWS will announce: 

Edit type and text 

Accessibility checkbox not checked (or checkbox checked) 

Accessibility 

Accessible format 

... 

When using the arrow keys in the Glossary, JAWS will announce: 

Glossary 

Edit, edit type and text 

Accessibility checkbox not checked (or checkbox checked) 

List of 55 items nesting level 1 

Accessibility 

Accessible format 

Accessible information and communication technology (ICT) 

… 

If we enter a letter (b) in the search box, JAWS will automatically announce how many items are in the search result list. This is good. 

B, 16 found 

If we enter a second letter (ba) JAWS will announce the updated results list. This is good. 

A, 2 found 

If we continue to arrow down then the next item in focus is the x which is announced as “unlabled x button, x found”. (“x” being the number of items in the list) What JAWS does not announce is that if we select this button, it will clear the text box. This is bad.

**Comment by MeD-Owner** [comment]
Adding a label to the checkbox and as placeholder text in the search box most likely fixes these issues. 
Then only difference is that using arrow keys announced the length of the list, and tabbing does not.

I would assume it's normal for the experience to be different based on how you navigate? (tab vs arrows)

**Comment by MeD-Owner** [comment]
Verified (for now)
